xbmc/guilib/GUIBaseContainer.cpp

```diff
@@ -1308,6 +1308,12 @@ std::string CGUIBaseContainer::GetLabel(int info) const
         label = StringUtils::Format("%u", numItems);
     }
     break;
+  case CONTAINER_ACTUAL_ITEMS:
```

Comment:
I dont think so, the way it is now only adds two lines of actual code (could even condense it into one).